addons/project/views/project_views.xml

```diff
@@ -1024,7 +1021,6 @@
             <field eval="2" name="priority"/>
             <field name="arch" type="xml">
                 <form string="Task" class="o_form_project_tasks" js_class="project_task_form">
-                    <field name="allow_subtasks" invisible="1" />
```

Comment:
We could keep it since it is invisible. 🙂

Reply:
Done